### ISSUE TITLE
AttributeError: 'Settings' object has no attribute 'IGNORE_DATES'

### ISSUE THREAD

**Issue by jovandeginste** [jonaswinkler/paperless-ng#299]
I upgraded my Docker image to v0.9.13 (jonaswinkler/paperless-ng:0.9.13), and noticed my documents were no longer being consumed (from the local consume directory).

I tried setting the `PAPERLESS_IGNORE_DATES` option, doesn't seem to fix it...

The full error:

```
INFO 2021-01-08 22:49:05,394 _sync Postprocessing...
22:49:12 [Q] ERROR Failed [2021-01-08_13-18-33.pdf] - 'Settings' object has no attribute 'IGNORE_DATES' : Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/django_q/cluster.py", line 436, in worker
    res = f(*task["args"], **task["kwargs"])
  File "/usr/src/paperless/src/documents/tasks.py", line 73, in consume_file
    override_tag_ids=override_tag_ids)
  File "/usr/src/paperless/src/documents/consumer.py", line 188, in try_consume_file
    date = parse_date(self.filename, text)
  File "/usr/src/paperless/src/documents/parsers.py", line 247, in parse_date
    date = __filter(date)
  File "/usr/src/paperless/src/documents/parsers.py", line 216, in __filter
    date.date() not in settings.IGNORE_DATES:
  File "/usr/local/lib/python3.7/site-packages/django/conf/__init__.py", line 83, in __getattr__
    val = getattr(self._wrapped, name)
AttributeError: 'Settings' object has no attribute 'IGNORE_DATES'
```

**Comment by jovandeginste** [comment]
Never mind, I was just stupid. It works perfectly, if I would just run it the correct way...

**Comment by jonaswinkler** [comment]
Please describe in more detail what you did.

**Comment by jovandeginste** [comment]
I was running a custom docker image for a while, to test my tika parser. For this, I had created a custom `settings.py`, which I mounted inside the container. I had not removed this file override, and so I was still using an old version (indeed, without the `IGNORE_DATES` attribute.

Since my parser was now part of the new image, I didn't need any of those customizations anymore, but had not deleted every trace...
